sparse/tests/test_coo.py

```diff
@@ -220,6 +220,20 @@ def test_reshape_same():
     assert s.reshape(s.shape) is s
 
 
+def test_reshape_function():
+    s = sparse.random((5, 3), density=0.5)
+    x = s.todense()
+    shape = (3, 5)
+
+    s2 = np.reshape(s, shape)
+    assert isinstance(s2, COO)
+    assert_eq(s2, x.reshape(shape))
+
+    # order parameter not actually supported
+    with pytest.raises(NotImplementedError):
+        np.reshape(s, shape, order='F')
```

Comment:
We might want to remove these two lines from the test, if we add this functionality later it'll crap out.

Reply:
An exception from coverage is already in place.

Reply:
Done.

I personally prefer to check these errors - in the past I've seen code that was supposed to raise a `DeprecationWarning` or `NotImplementedError` fail to actually raise due to a bug. I don't think of the tests as the-one-uneditable-source-of-truth, I think someone adding this functionality wouldn't be surprised to see a test fail due to a `NotImplementedError` not being raised anymore, and be fine changing the test.

Anyway, happy to cede to you here. I believe this should be good to go.

Reply:
Fair enough. 👍